src/components/cart/cart-item/cart-item.tsx

```diff
@@ -67,12 +70,42 @@ export const CartItem = ({
                 <div className={styles.productInfo}>
                     <div className={styles.productNameAndPrice}>
                         <div className={styles.productName}>{productName}</div>
+
                         {item.fullPrice?.formattedConvertedAmount && (
                             <ProductPrice
                                 price={item.fullPrice?.formattedConvertedAmount}
                                 discountedPrice={item.price?.formattedConvertedAmount}
                             />
                         )}
+
+                        {item.descriptionLines && item.descriptionLines.length > 0 && (
+                            <div className={styles.options}>
```

Comment:
maybe this collapsable container deserves some generic component?
or at least a separate `CartItemOptions` component